src/to_do/deque_list/__init__.py

```diff
@@ -0,0 +1,57 @@
+import typing
+from dataclasses import dataclass, field
+
+from src.to_do.base_list import BaseListTask
+from collections import deque
+from src.to_do.deque_list.task import DequeTask, task_01, task_02, task_03
+
+
+@dataclass
+class DequeList(BaseListTask):
+    tasks: typing.Deque[DequeTask] = field(default_factory=deque)
+
+    def add_task(self, task: DequeTask) -> None:
+        self.tasks.append(task)
+        task.status = 'PENDING'
```

Comment:
Generalnie nie ma to znaczenia w tym przypadku, ponieważ obiekty są modyfikowalne, ale najlepiej byłoby zmienić status zadania i dopiero wrzucać go na listę 😉